configure.ac

```diff
@@ -1403,6 +1403,8 @@ AC_SUBST(ENABLE_SETCAP)
 
 AC_CONFIG_LINKS([discovery.xml:discovery.xml])
 AC_CONFIG_LINKS([browser/package.json:browser/package.json])
+AC_CONFIG_LINKS([browser/.eslintrc:browser/.eslintrc])
+AC_CONFIG_LINKS([browser/.prettierrc:browser/.prettierrc])
```

Comment:
Should we be linking .beforeprettier here too?

Reply:
yes, but prefer to check documentation first

Reply:
so, I've read the documentation, I'm still not entirely sure why .eslintrc/.prettierrc are included and .eslintignore/.prettierignore/.beforeprettier are not, is there something I'm missing here?

Reply:
nevermind, maybe this is fine, we do `--ignore-path $(srcdir)/.beforeprettier \` and the like so that includes the srcdir there... alright, I think this is fine then